### DISCUSSION TITLE
Shadcn-UI for Mobile Apps

### DISCUSSION THREAD

**Discussion by ibnsina77** [shadcn-ui/ui#1841] [Q&A]
Hello,

may I ask you whether Shadcn UI might be used for mobile app development?

**Comment by jacob-leger** [comment]
Do you mean with _React Native_, _Flutter_, or with native _Android_ and _IOS_ code, namely _Kolin_ and _Swift_?

I'm not sure about _React Native_, but everything else is a **solid NO**, sorry.

Why no? **Shadcn/UI** needs to use _RadixUI_, which needs React. So if you're using _React Native_, **my answer is maybe**. I have never worked with it myself.

Sorry if this didn't help, but this has been unanswered for almost a week so I thought I might give it a shot.

**Reply by ScarAnd** [reply]
You can actually use shadcn. But you need to use tauri as a framework. Tauri is based on react + vite in the frontend and rust in backend. Then you are able to use shadcn

**Reply by jacob-leger** [reply]
I forgot about that one!

**Comment by tamacoconutlab** [comment]
can try this https://reactnativereusables.com
